src/util/log/fd_log.c

```diff
@@ -1185,6 +1208,16 @@ fd_log_private_halt( void ) {
   fd_log_private_app[0]         = '\0';
   fd_log_private_app_id         = 0UL;
 
+  if( FD_LIKELY( fd_log_private_shared_lock!=fd_log_private_shared_lock_local ) ) {
+    /* Note: this will not unmap this in any clones that also inherited
```

Comment:
The comment isn't strictly true, if it was cloned with `CLONE_VM` then the unmap is shared, but the contract of `clone()` is that the caller is already expected to understand and handle the free/cleanup semantics.

Reply:
Line 684.  More theoretical than anything but easy to cleanup while in there.